include/GafferUI/ConnectionCreator.h

```diff
@@ -63,10 +58,14 @@ class ConnectionCreator : public Gadget
 		/// and tangent for the connection as the drag progresses within the destination.
 		virtual void updateDragEndPoint( const Imath::V3f position, const Imath::V3f &tangent ) = 0;
 
+		/// May be called by the recipient of a drag to figure out if this
+		/// ConnectionCreator can set up the connection.
+		virtual bool canCreateConnection( const GafferUI::Nodule *endpoint ) = 0;
```

Comment:
Where possible I like to order methods in the order of anticipated use. This makes them a bit easier to document and understand. Since the first thing a drag recipient will do is call `canCreateConnection()`, it would be a tiny bit clearer to put this before `updateDragEndPoint()`.